project/Dependencies.scala

```diff
@@ -17,61 +17,61 @@ object Dependencies {
   val scala3Version = "3.1.2" // not yet enabled - missing pekko-http/pekko-management Scala 3 artifacts
   val scalaVersions = Seq(scala212Version, scala213Version)
 
-  val akkaVersion = System.getProperty("override.akka.version", "2.6.20")
-  val akkaVersionInDocs = akkaVersion.take(3)
+  val pekkoVersion = System.getProperty("override.pekko.version", "0.0.0+26599-83545a33-SNAPSHOT")
+  val pekkoVersionInDocs = pekkoVersion.take(3)
   val cassandraVersionInDocs = "4.0"
-  // Should be sync with the version of the driver in Alpakka Cassandra
+  // Should be sync with the version of the driver in Pekko Connectors Cassandra
   val driverVersionInDocs = "4.6"
 
-  val alpakkaVersion = "3.0.4"
-  val alpakkaVersionInDocs = alpakkaVersion.take(3)
+  val pekkoConnectorsVersion = "0.0.0+54-ee23f1bc-SNAPSHOT"
+  val pekkoConnectorsVersionInDocs = pekkoConnectorsVersion.take(3)
```

Comment:
could we use `pekkoConnectorsVersionInDocs = "current"` for now? Likewise with pekkoManagementVersion.

Reply:
Missed this, yes we should use `current`